src/target/ext/offsets.rs

```diff
@@ -0,0 +1,18 @@
+//! Handle the `qOffsets` command
+
+use crate::target::Target;
+
+/// Target Extension - Handle the `qOffsets` command
+pub trait OffsetsCmd: Target {
+    /// Handle the `qOffsets` command
+    ///
+    /// For some targets, sections may be relocated from their base address. As
```

Comment:
Could you push these docs up to the module level? It would make it easier to understand what this extension does while browsing through the generated docs.